notebooks/timeframes.ipynb

```diff
@@ -0,0 +1,729 @@
v9ttjDZz3pdqXt5iZYNDWCpz87wr6cUqtOVqFXn9uSklHCpMg+VmlopqXP5uB0d9xDgoZYFEjqDUbeTs3miqgQRvTv2QQHlmp3hb+3B6sWJtA3wIvFb+4h91yNXXTb42+bjvPpT/k8fs1obowPs5tuR2ilrWefb00YQqCPB6utfNetZ59byCurJedcDVOs1MZVS5/NwekCd4OxgQZjg9nrf/bTGYqr6q3Sc9tSbXPoH+jDm4sm0WiULFizm9Lqi/dvC93zeTsli9d/OMVdk8N54PJhdtPtDK209eyzv7cHtycO4atDZ8krq7Wbri0xVzv1lHXnl9TSZ3NwusCdlp9GWr55s+9IKVm1PZNRAwKYNqLnDWks0baEEf0DeGNBAqdLa7lvbdpFrTxtpQvwzZECnvr0MFeN6c/Ts6MvqG+3pW5XaKWtd58XTI1ASsnalCy76toKc7VTM0oI9vNk9MBAu+pqhdMFbktIySjh6JkKFid1b8CNPUmM6MPffxnP3pxS/uv9A5hMtq/x3p9TykPv7iM2LIhX7hjvtL2NnYnBvf24NmYQ7+7KcYlZllpIzSzhEivltx0Bl/5PXL09iz7+XswdH6a1KWZx/bhBLLtuDF+mn2X5l0dtqpVdUs29b6XRP9CHVQsT8fNyunmlnZbF0yKoqDM4TbvgrjhdWkPuuVqnnaasPVw2cGcVV7PlWAHzLgl3qBla7r10GIuSIli1PdNmpV8lVfUsWL0bk5S8tXgS/QJcY2JjZ2FCeG/ihgSzekeWXb6ZaY2zzy/ZHi4buN/cmYWHm2D+5KFam2IxT14/lmuiB/LsF0f4Kv2MVfdd22Dk3rVpnCmv440FiTbrOKewHU29uiPILK7m++OFWptjc1IzSujt50nUAOvktx0Bp/v+a047xvLaRt5Py2V2XCj9e/nYVdsauLsJXro9nnlv7OKhd/dSF/QceXU7GRo0lOUzlnd7IIzRJHn4vf0cyC3jtXkTu+yQ6EjtPh1d11Lt62IH8fxXx1i9I9Psya6toWttzNFOOVXCJZF9rZrf1ntbV6cL3OY0P1+/J4eaBmO3em73VNta+Hi6Myshi1251YjS3+DhnU12eTbJnyUDWBy8pZT85fMjbD5SwNOzx3JNzMAut9Gy0byWjf21whJtT3c37p4SwQtfH+PomYoedcvTs8+552rIK6sl+bJhna5nbV2tcbpUSW1jLbWNHdewGowm3tqZzSWRfaw+vVZX2tbmrzv+QIHnH5HUEtKwDDcZTE1jDcu2WN5e9Y0fM3lzZxb3XRrJQjM/0Oztrx60HcnnOyYNwdfTvce9uvXss7X6b1uqqzVOF7j3n93P/rP7O3x/0+EC8spqWWyFATeWalubnPIcDG4FFHg/hVGco3/9nxDS2+L2qp/9lM/yL49y/bhBLL12jNnb2dtfPWg7ks/Bfl78YmIYGw/kU1xVbzdda9KVdkpGCX38vRjZw1HPlupqjdMF7q5YvSOT8D5+XNXDvJ8eaGmjanA7TZnHO3jJ4fRreJzwXhFm72NXRgm/f/8nJkX04b9vjXOZOlhXYVFSJA0GE++kZmttitWRUrIr4xyTh7lO/XYLLhW4D+SWsTe7lEVJEU7RaH35jOX4eTZNaNzgfoxznq/hZ5pEYuCLZk3C8HNhJfetTWNIH19W3j3RocoiFeYxPCSA6VEhvJOaTb1Bv1NxdYfTpbXklblW/XYLLhW4V23PJNDbg1sT9Ns8xhLmxc5j5eyVDPBv+vbQt+9RLo+uY8/P3vxz66lOty2sqGPB6j14e7rz5qJJumhnq7AN90wbRnFVA58eyNfaFKuSYuX+JI6E01WVdMSZ8lq+TD/DoqkRBHg7j9vzYucRGdyUr586ZCpSSn63/gArNh0nNNiHm8YPvmibqnoDi97cQ2lNA+/fP4UhffzsbbbCjiSN6EvUgEBWbc/klomDdd/ewVxSM0roa4P8tiPgPBGsmeG9h7e7/K2d2UgpWTA1wu7atuZ8XSEEf7sljoKKeh778CADAn2Yel4DrUajiQfX7ePY2UreWJDQo8oarfzVUtsRfRZCsHhaBI9/lE5KRglTh1vWUE2PPkspSckoYfKwvjb5INLSZ3MQXeVChRA+wDbAm6ZA/6GU8qnOtklISJBpafrprFXTYGDKc98xdXhfXrtrotbm2IXy2kZ++XoK+WW1/Gr6cNal5pBfVouvlzs1DUaevzmW23UwTZvCPtQ1Gpn4l28wmiT1BhOhwb4smRXlMH162pJdUs3lK7bylxujmT8lQmtzrIIQYq+UMsGcdc3JcdcDV0op44B44BohxOSeGGhLqhqqqGqoumDZR/vyKK9ttErPbUu17UF7ukG+nqxZlIgQ8Levj5NXVosEahqMeLgJqzyI1MpfLbUd1eevD52l3mCizmBC0jTxwNIN6Wzcn2dT3Z7Skbat55fU0mdzMGeyYCmlbPHAs/lHt51rDhYc5GDBwdbXJpNkzfZMxg0O6nIIt7W17UVHuqHBvu0GaINJsmLTcZvp2gO9HWu9a6/YdBxDm4ZTtY1Gs64DPfqccqqEfgFeDA+xTX5bS5/NwayqEiGEuxDiAFAIfCOl3NXOOslCiDQhRFpRUZG17ew2P5woIqO4mnum6b/nti0oqmx/4EW+FWdIUeifjs63I14HUkpSM85xiY3y246AWYFbSmmUUsYDg4FJQoiYdtZZKaVMkFImhISEWNvObrNqeyYDenlzXewgrU3RhNBgX4uWK5yTjs53gI8HtQ2OVd+dXVLD2Yo6q80v6YhYVMctpSwDvgeusY051uXY2Qq2/1zM3VMi8HTR2VuWzIrCt026xNfTnSWzojSySKEF7V0HbgIq6wxc+d9b+eRAnlmDtvRAio36kzgSXUYzIUSIECK4+W9f4GrgmK0NswZrtmfh4+nGvEtct3pi7vgwnrs5lrBgXwQQFuzLczfHOmw1gaJ7tHcd/M8v41mfPJm+AV48/N4BbvrnTvbllGptapekZpQQEu
```

Comment:
'not necessarily leads' --> 'doesn't necessarily lead'

Reply:
'might lose some information' is this because we chop out some information at the start/end? Intuitively I feel like we should always go back to the same interpolated line but maybe I'm just wrong and need to do actually do the maths?

Reply:
Done
Added: "This is because of possibly larger period lengths, overlapping of periods (both leading to averages taken over several data points possibly losing their information), and chopped-off information at the beginning and the end of the timeseries for shifted periods."